pkg/ansiescape/split.go

```diff
@@ -0,0 +1,89 @@
+package ansiescape
+
+import "bytes"
+
+// dropCR drops a terminal \r from the data.
+func dropCR(data []byte) []byte {
```

Comment:
Nevermind I misread my own code, it will drop both since it does not immediately return